examples/gpt/schedule.py

```diff
@@ -416,11 +415,12 @@ def order_args_fn(*args, **kwargs):
 
 
 def broadcast_input(sch):
-    group_src_rank = _get_global_rank(sch.group, 0)
+    group_src_rank = dist.get_global_rank(sch.group, 0)
```

Comment:
@szhengac please double check.

Reply:
APIs start with underscore, is not encouraged to use. APIs in the `torch.distributed` package are routed to the `torch.distributed.distributed_c10d`